ext/OceananigansNCDatasetsExt.jl

```diff
@@ -57,48 +57,74 @@ const BuoyancyBoussinesqEOSModel = BuoyancyForce{<:BoussinesqSeawaterBuoyancy, g
 ##### Extend defVar to be able to write fields to NetCDF directly
 #####
 
+
+function squeeze_reduced_dimensions(data, reduced_dims)
+    # Fill missing indices from the right with 1s
+    indices = Any[:,:,:]
```

Comment:
```suggestion
    indices = Any[:, : ,:]
```

what does this mean exactly?

Reply:
I can (and in fact I probably will do it regardless), but that doesn't affect calculations with any of the software (that I'm aware of) which reads NetCDF. So broadcasting and other useful things wouldn't work. Without the singleton dimensions (i.e. with the change that I'm exploring here), those things will work out of the box for most software.

Also if we include the singleton reduced dimension, we still would need to pick a value for that length-1 coordinate. In Oceananigans, that value is `nothing` I think, but there's no analog to that for NetCDF, so we'd need to pick an arbitrary value like 1 or 0, which I'd like to avoid.